### DISCUSSION TITLE
BUGS

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#98] [General]
A place for everyone to talk about found / discovered bugs.

**Comment by WilsonBradley** [comment]
Wall blocking not enabled at starting section / hallway.
Not a high priority,
 but able to look through walls on left when facing forward.
See video
https://youtu.be/SaXFkTnYLmc

**Reply by curif** [reply]
Sometimes unity starts with the player not centered. To solve a workaround is to long press the menu button (rigth controller)

**Comment by WilsonBradley** [comment]
Cabinets are to low. 
Higher priority.
In this example, while standing and using guardian (not disabled), while standing most arcades, seem to short. 
Eye level is the height where the finger is pointing.
![com curif AgeOfJoy-20230206-184016](https://user-images.githubusercontent.com/8173893/217127699-fa76c45a-201a-4c3a-b9c2-f88d7df4a89c.jpg)

**Comment by WilsonBradley** [comment]
Ambient volume to high. Best if able to somehow adjust.
This is a problem when recording and trying to explain things in video.

**Reply by curif** [reply]
Move this to the config suggestion card.

**Comment by WilsonBradley** [comment]
Make the NPC's/ AVatars leave when you put money in a Arcade. They seem to stick around with their arms in the way.

**Comment by WilsonBradley** [comment]
For some reason on Galaga game - In first room on right  , you can't hear the explosions when ship is destroyed. The second one in other room you can hear it.

**Comment by WilsonBradley** [comment]
Cabinet CRT emulation screens are to low within Cabinet while playing. 
Centipede 
Kangaroo 
and a few others could be aligned better within Cabinet.
Once a good placement is found/ just use same if cabinet is near same
